spaghetti-haxe-support/src/test/groovy/com/prezi/spaghetti/haxe/impl/HaxeModuleProxyGeneratorVisitorTest.groovy

```diff
@@ -5,28 +5,29 @@ import com.prezi.spaghetti.generator.ModuleGeneratorSpecification
 class HaxeModuleProxyGeneratorVisitorTest extends ModuleGeneratorSpecification {
 	def "generate"() {
 		def definition = """
-module com.example.test
+module com.example.test {
 
-interface MyInterface<T> {
+	interface MyInterface<T> {
+		/**
+		 * This should have nothing to do with the results.
+		 */
+		someDummyMethod(x: int): void;
+	}
 	/**
-	 * This should have nothing to do with the results.
+	 * Does something.
 	 */
-	void someDummyMethod(int x)
```

Comment:
I think the diff is just confusing, this is the interface method, it's added in the new syntax a few lines above.